### PR TITLE
feat: create aboutus modal

### PR DESCRIPTION
**Pull request by flooksan**
Pls. review

### CODE REVIEW

File: src/components/AboutUs/AboutUs.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  width: 600,
+  bgcolor: 'background.paper',
+  borderRadius: '10px !important',
+  boxShadow: 24,
+  p: 4,
+  color:'black',
+};
```
Comment (Khongchai) [inline src/components/AboutUs/AboutUs.jsx:19]: Does this have to be outside of the function?

Comment (flooksan) [reply src/components/AboutUs/AboutUs.jsx:19]: Default on mui modal example them define an outside function component, but in best practice, I don't know whether outside or inside are best.

Comment (Khongchai) [reply src/components/AboutUs/AboutUs.jsx:19]: Ok. I'll resolve this for now. This is a minor style-related issue. We can refactor later.

Comment (flooksan) [reply src/components/AboutUs/AboutUs.jsx:19]: Ok,Thanks krub P'Khong

**Pull request merged**